Userland/Libraries/LibArchive/Tar.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
```

Comment:
The original file has

```
 * Copyright (c) 2020, Peter Elliott <pelliott@ualberta.ca>
 * Copyright (c) 2021, Idan Horowitz <idan.horowitz@serenityos.org>
```

Please don't move code without preserving the relevant copyright header lines.

Reply:
Aw crap, I didn't pay attention to that... I'll amend the PR to fix this.